### PR TITLE
Generate all endpoints under the `/acs/systems` route

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
wrt https://github.com/seamapi/docs/issues/268

image

### CODE REVIEW

File: src/lib/reference.ts [hunk 1]
```diff
@@ -26,6 +29,14 @@ export const reference = (
   }
 
   for (const route of metadata.routes ?? []) {
+    const k = `api-clients${route.path.replace('acs', 'access-control-systems')}/README.md`
```
Comment (razor-x) [inline src/lib/reference.ts:32]:
```suggestion
    const k = `api${route.path}/README.md`
```

File: src/lib/template-context.ts [hunk 2]
```diff
@@ -42,36 +76,52 @@ export function setEndpointTemplateContext(
...
 
-export interface EndpointTemplateContext {
-  description: string
-  title: string
-  path: string
-  request: {
-    preferredMethod: string
-    parameters: Array<{
+export interface ResourceTemplateContext {
+  resource: {
```
Comment (razor-x) [inline src/lib/template-context.ts:80]: We may need to support more than one resource.

File: src/lib/template-context.ts [hunk 3]
```diff
@@ -1,4 +1,38 @@
...
+      description: string
+    }>
+  }
+  response: {
+    description: string
+    resourceType: string | null
+    properties: null | Array<{
+      name: string
+      description: string
+    }>
```
Comment (razor-x) [inline src/lib/template-context.ts:21]: We can remove this now.

**Review by razor-x — APPROVED** [review]

File: docs/api-clients/access-control-systems/systems/README.md
Comment (razor-x) [inline docs/api-clients/access-control-systems/systems/README.md:1]: We should not be modifing this page. We should generate to `/docs/api/acs/systems/README.md`

**Comment by razor-x** [comment]
@andrii-balitskyi I spoke with @DebbieAtSeam and updated https://github.com/seamapi/docs/issues/299 with the content / structure we want for the docs pages. It's pretty close to what you have now. Let's get that format on the beta branch and we can iterate from there.

For `/acs/systems/README.md`, we can merge what you have on this PR and follow up in the next PR to match the format I added to https://github.com/seamapi/docs/issues/300

File: src/lib/template-context.ts (hunk 2 above)
Comment (andrii-balitskyi) [reply src/lib/template-context.ts:80]: what are the use cases for this?

File: src/lib/template-context.ts (hunk 3 above)
Comment (andrii-balitskyi) [reply src/lib/template-context.ts:21]: Not sure I understand why, could you explain?

File: docs/api-clients/access-control-systems/systems/README.md
Comment (andrii-balitskyi) [reply docs/api-clients/access-control-systems/systems/README.md:1]: 2381d39151a9d251722770107192cc52da79cb8d

File: src/lib/template-context.ts (hunk 2 above)
Comment (razor-x) [reply src/lib/template-context.ts:80]: For example, /devices/README.md will likely continue device and device_provider

File: src/lib/template-context.ts (hunk 3 above)
Comment (razor-x) [reply src/lib/template-context.ts:21]: Endpoints won't document response properties since those are documented by the resource.

Comment (andrii-balitskyi) [reply src/lib/template-context.ts]: 38dd709ed12cacf2ae5e664d1141924e712ba7f3

File: src/lib/template-context.ts (hunk 2 above)
Comment (andrii-balitskyi) [reply src/lib/template-context.ts]: 508cf4a808efba29da4cbe1da78dcaf63bc9bc0d

**Comment by razor-x** [comment]
How is this PR related to https://github.com/seamapi/docs/pull/311? Do we need to merge that one first?

**Comment by andrii-balitskyi** [comment]
closing in favour of https://github.com/seamapi/docs/pull/314

**Pull request closed without merging**